src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
```

Comment:
Creating a new `Random` object each time the method is called is not efficient. Consider making it a private class-level variable so that it's created only once for the instance of the `Lottery` class.